src/components/Tokens/constants.ts

```diff
@@ -1,5 +1,6 @@
 import { ChainId } from '@uniswap/sdk-core'
 
+// Breakpoints specifically for the token pages
```

Comment:
outside of the realm of this PR but I would love for these to be deprecated in favor of our sitewide breakpoints